jellyfin-api/src/commonMain/kotlin/org/jellyfin/sdk/api/client/util/UrlBuilder.kt

```diff
@@ -64,7 +69,11 @@ public object UrlBuilder {
 
 					lastEnd = i
 				}
+
```

Comment:
If these aren't intended, you could set `ij_kotlin_line_break_after_multiline_when_entry = false` in the `.editorconfig` if you like.

Reply:
I actually prefer those so they will slowly get into the codebase :p